sdk/textanalytics/Azure.AI.TextAnalytics/src/PiiEntity.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// A word or phrase identified as a Personally Identifiable Information
+    /// that can be categorized as known type in a given taxonomy.
+    /// The set of categories recognized by the text analytics service is described at
+    /// <a href="https://aka.ms/tanerpii"/>.
+    /// </summary>
+    public readonly struct PiiEntity
+    {
+        internal PiiEntity(Entity entity)
+        {
+            Category = entity.Category;
+            Text = entity.Text;
+            SubCategory = entity.Subcategory;
```

Comment:
In python we have this as one word, which would translate to `Subcategory`. I think it's better to keep it this way, since that's what the service has, and we have also had this convo in FR (about [submodels](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py#L596)) and we decided to make them one word

Reply:
Yeah, when I was doing this I looked at how TA .NET was doing it and we didn't make the change (see [CategorizedEntity](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/CategorizedEntity.cs#L48)) so I am keeping it as is here for consistency with the library.

Reply:
oh i see, makes sense